client/vue.config.js

```diff
@@ -28,5 +28,16 @@ module.exports = {
     'vuetify',
     'vuex-module-decorators'
   ],
-  publicPath: '/namerequest'
+  publicPath: '/namerequest',
```

Comment:
Hmm.  Not sure about this one.  Seems a little strange.  I will ask Scott Rumsby to review.

Reply:
Sounds good. The proxy is to avoid CORS error when running locally.